libs/domains/organizations/data-access/src/lib/domains-organizations-data-access.ts

```diff
@@ -18,6 +25,58 @@ export const organizations = createQueryKeys('organizations', {
       return response.data
     },
   }),
+  gitTokens: (organizationdId) => ({
```

Comment:
types are missing and we should always use object to pass params
```suggestion
  gitTokens: ({ organizationdId } : { organizationdId: string }) => ({
```
Valid for all methods here

Reply:
Fixed https://github.com/Qovery/console/pull/913/commits/5f0ff27fbf70a12706bd3d915ea9360cbec3cc55